examples/usecases/ecommerce-session-based-next-item-prediction-for-fashion.ipynb

```diff
@@ -0,0 +1,1546 @@
+{
```

Comment:
<p>lets remove dataset = nvt.Dataset(cudf.concat([train.to_ddf().compute(), valid.to_ddf().compute()]))</p><p> </p><p>and we use workflow.fit(train)</p><p> </p><p>Lets add a description above.</p><p> </p><p>"We call fit and transform similar to the scikit learn API.</p><p> </p><p>Categorify will map unkown item_ids (and purchase_ids) to a special category <code>0</code> in the validation dataset. This can bias the validation metrics. In our example, almost all item_ids in validation are available in train and we neglect it. </p><p>"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/models/pull/664/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='29'/>